code/modules/events/shuttle_loan.dm

```diff
@@ -175,8 +175,11 @@
 				shuttle_spawns.Add(/obj/item/pizzabox/meat)
 				shuttle_spawns.Add(/obj/item/pizzabox/meat)
 				shuttle_spawns.Add(/obj/item/pizzabox/vegetable)
-				if(prob(10))
-					shuttle_spawns.Add(/obj/item/pizzabox/bomb)
+				if(prob(20))
+					if(prob(50))
+						shuttle_spawns.Add(/obj/item/pizzabox/bomb)
+					else
+						shuttle_spawns.Add(/obj/item/pizzabox/margherita/robo)
```

Comment:
It'd probably be a decent idea to convert this all to 6 picks from a weighted list

Reply:
Just make it pure RNG sissies

```dm
//tab to be under if(PIZZA_DELIVERY)
var/naughtypizza = list(/obj/item/pizzabox/bomb,/obj/item/pizzabox/margherita/robo) //oh look another blaklist, for pizza nonetheless!
var/nicepizza = (subtypesof(/obj/item/pizzabox) - naughtypizzas)
var/gimmepizza
for(var/i in 1 to 6)
    gimmepizza = pick(nicepizza)
    shuttle_spawns.Add(gimmepizza)
if(prob(20))
    gimmepizza = pick(naughtypizza)
    shuttle_spawns.Add(gimmepizza)
```

Reply:
That's 6 pizzas+However many naughtypizzas were picked, so you immediately know how many borked pizzas you got.